src/core/lib/json/json_reader.cc

```diff
@@ -279,10 +279,7 @@ JsonReader::Status JsonReader::Run() {
       case GRPC_JSON_READ_CHAR_EOF:
         if (IsComplete()) {
           return Status::GRPC_JSON_DONE;
-        } else {
-          return Status::GRPC_JSON_PARSE_ERROR;
-        }
-        break;
+        return Status::GRPC_JSON_PARSE_ERROR;
```

Comment:
Missing a `}`.

Reply:
Alternatively, you could write `return IsComplete() ? Status::GRPC_JSON_DONE : Status::GRPC_JSON_PARSE_ERROR;`

Reply:
Thanks, fixed.

🤦‍♂️